autoinstrumentation/python/requirements.txt

```diff
@@ -1,61 +1,61 @@
+opentelemetry-propagator-jaeger==1.26.0
 opentelemetry-propagator-aws-xray==1.0.1
 
-opentelemetry-instrumentation==0.46b0
-opentelemetry-propagator-ot-trace==0.46b0
+opentelemetry-instrumentation==0.47b0
+opentelemetry-propagator-ot-trace==0.47b0
 
 # Copied in from https://github.com/open-telemetry/opentelemetry-python-contrib/tree/main/instrumentation
-opentelemetry-instrumentation-aio-pika==0.46b0
-opentelemetry-instrumentation-aiohttp-client==0.46b0
-opentelemetry-instrumentation-aiopg==0.46b0
-opentelemetry-instrumentation-asgi==0.46b0
-opentelemetry-instrumentation-asyncio==0.46b0
-opentelemetry-instrumentation-asyncpg==0.46b0
-opentelemetry-instrumentation-aws-lambda==0.46b0
-opentelemetry-instrumentation-boto==0.46b0
-opentelemetry-instrumentation-boto3sqs==0.46b0
-opentelemetry-instrumentation-botocore==0.46b0
-opentelemetry-instrumentation-cassandra==0.46b0
-opentelemetry-instrumentation-celery==0.46b0
-opentelemetry-instrumentation-confluent-kafka==0.46b0
-opentelemetry-instrumentation-dbapi==0.46b0
-opentelemetry-instrumentation-django==0.46b0
-opentelemetry-instrumentation-elasticsearch==0.46b0
-opentelemetry-instrumentation-falcon==0.46b0
-opentelemetry-instrumentation-fastapi==0.46b0
-opentelemetry-instrumentation-flask==0.46b0
-opentelemetry-instrumentation-grpc==0.46b0
-opentelemetry-instrumentation-httpx==0.46b0
-opentelemetry-instrumentation-jinja2==0.46b0
-opentelemetry-instrumentation-kafka-python==0.46b0
-opentelemetry-instrumentation-logging==0.46b0
-opentelemetry-instrumentation-mysql==0.46b0
-opentelemetry-instrumentation-mysqlclient==0.46b0
-opentelemetry-instrumentation-pika==0.46b0
-opentelemetry-instrumentation-psycopg==0.46b0
-opentelemetry-instrumentation-psycopg2==0.46b0
-opentelemetry-instrumentation-pymemcache==0.46b0
-opentelemetry-instrumentation-pymongo==0.46b0
-opentelemetry-instrumentation-pymysql==0.46b0
-opentelemetry-instrumentation-pyramid==0.46b0
-opentelemetry-instrumentation-redis==0.46b0
-opentelemetry-instrumentation-remoulade==0.46b0
-opentelemetry-instrumentation-requests==0.46b0
-opentelemetry-instrumentation-sklearn==0.46b0
-opentelemetry-instrumentation-sqlalchemy==0.46b0
-opentelemetry-instrumentation-sqlite3==0.46b0
-opentelemetry-instrumentation-starlette==0.46b0
-opentelemetry-instrumentation-system-metrics==0.46b0
-opentelemetry-instrumentation-threading==0.46b0
-opentelemetry-instrumentation-tornado==0.46b0
-opentelemetry-instrumentation-tortoiseorm==0.46b0
-opentelemetry-instrumentation-urllib==0.46b0
-opentelemetry-instrumentation-urllib3==0.46b0
-opentelemetry-instrumentation-wsgi==0.46b0
+opentelemetry-instrumentation-aio-pika==0.47b0
+opentelemetry-instrumentation-aiohttp-client==0.47b0
+opentelemetry-instrumentation-aiohttp-server==0.47b0
+opentelemetry-instrumentation-aiopg==0.47b0
+opentelemetry-instrumentation-asgi==0.47b0
+opentelemetry-instrumentation-asyncio==0.47b0
+opentelemetry-instrumentation-asyncpg==0.47b0
+opentelemetry-instrumentation-aws-lambda==0.47b0
```

Comment:
I think we ~can~ should drop this, as it will probably break everything https://github.com/open-telemetry/opentelemetry-python-contrib/issues/2744

Reply:
Thanks @xrmx , you're right. `aws-lambda` instrumentation should be a part of the auto-instrumentation image. This instrumentation works only in AWS Lambda